packages/react-vapor/src/components/navigation/examples/NavigationConnectedExamples.tsx

```diff
@@ -8,7 +8,8 @@ import {NavigationConnected} from '../NavigationConnected';
 const navigationConnectedExampleId = 'navigation-connected';
 const navigationConnectedExampleLoadingIds = ['loading-' + navigationConnectedExampleId];
 
-export class NavigationConnectedExamples extends React.Component<any, any> {
+export class PaginationConnectedExamples extends React.Component<any, any> {

```

Comment:
The file should also be renamed I think. Also you shouldn't need the `<any, any>`

```suggestion
export class PaginationConnectedExamples extends React.PureComponent {
```